espnet2/tasks/svs.py

```diff
@@ -296,7 +298,7 @@ def optional_data_names(
         cls, train: bool = True, inference: bool = False
     ) -> Tuple[str, ...]:
         if not inference:
-            retval = ("spembs", "durations", "pitch", "energy", "sids", "lids")
+            retval = ("spembs", "durations", "pitch", "energy", "sids", "lids", "feats")
```

Comment:
Probably, we do not need to add additional feats here but reuse the singing variable. We can have more discussion after the deadline for interspeech.